docs/source/using_the_app.rst

```diff
@@ -58,19 +58,24 @@ and run:
 
 Once the app launches you will begin to see printouts in the terminal.
 Navigate to the app output by clicking on the "port" plugin on the right-hand tool bar
-(see image below), type in port number 7501, and click "update".
+(see image below), In the Port Viewer, select the "LP Port" and ensure it is set to port 7501 
+before proceeding. After confirming the port, click on it and allow the app to load.
 
-.. image:: https://imgur.com/0XxDcpZ.png
+.. image:: https://imgur.com/a/e4DvzGA
```

Comment:
add .png (so `.. image:: https://imgur.com/a/e4DvzGA.png`). this is necessary to render the image in readthedocs

Reply:
image looks great!